pkg/bwagreement/test/common.go

```diff
@@ -36,12 +36,20 @@ func GeneratePayerBandwidthAllocation(action pb.PayerBandwidthAllocation_Action,
 		return nil, err
 	}
 
+	// generate expiration date
+	var exp int64
+	if expired {
+		exp = time.Now().AddDate(0, 0, -5).Unix()
+	} else {
+		exp = time.Now().Add(time.Hour * 24 * 10).Unix()
```

Comment:
expires in 10 days?

Reply:
Yeah, that's the test was using before I added the ability to generate an expired date